ScriptingExample/src/test/java/org/openzen/zenscript/scriptingexample/tests/actual_test/arithmethic_operators/AdditionsAndSubtractionTests.java

```diff
@@ -0,0 +1,93 @@
+package org.openzen.zenscript.scriptingexample.tests.actual_test.arithmethic_operators;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.openzen.zenscript.scriptingexample.tests.helpers.ScriptBuilder;
+import org.openzen.zenscript.scriptingexample.tests.helpers.ZenCodeTest;
+
+import java.util.*;
+import java.util.stream.Stream;
+
+public class AdditionsAndSubtractionTests extends ZenCodeTest {
```

Comment:
What better way to add many tests than to parameterize them 😉 
11 numeric types (left operand) * 11 numeric types (right operand) * 2 (addition and subtraction) = 242 Tests, most of which currently fail...